cpp/src/barretenberg/stdlib/encryption/ecdsa/ecdsa.test.cpp

```diff
@@ -57,4 +57,97 @@ TEST(stdlib_ecdsa, verify_signature)
     bool proof_result = verifier.verify_proof(proof);
     EXPECT_EQ(proof_result, true);
 }
+
+TEST(stdlib_ecdsa, verify_signature_noassert_succeed)
+{
+    Composer composer = Composer();
+
+    // whaaablaghaaglerijgeriij
+    std::string message_string = "Instructions unclear, ask again later.";
+
+    crypto::ecdsa::key_pair<curve::fr, curve::g1> account;
+    account.private_key = curve::fr::random_element();
+    account.public_key = curve::g1::one * account.private_key;
+
+    crypto::ecdsa::signature signature =
+        crypto::ecdsa::construct_signature<Sha256Hasher, curve::fq, curve::fr, curve::g1>(message_string, account);
+
+    bool first_result = crypto::ecdsa::verify_signature<Sha256Hasher, curve::fq, curve::fr, curve::g1>(
+        message_string, account.public_key, signature);
+    EXPECT_EQ(first_result, true);
+
+    curve::g1_bigfr_ct public_key = curve::g1_bigfr_ct::from_witness(&composer, account.public_key);
+
+    std::vector<uint8_t> rr(signature.r.begin(), signature.r.end());
+    std::vector<uint8_t> ss(signature.s.begin(), signature.s.end());
+
+    stdlib::ecdsa::signature<Composer> sig{
+        curve::byte_array_ct(&composer, rr),
+        curve::byte_array_ct(&composer, ss),
+        27,
```

Comment:
Can use `signature.v` instead of 27.

Reply:
Good point - It is only in the dsl folder that this needs to be 27 atleast until we change the ecdsa structure to have a v parameter